modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -88,13 +86,19 @@ public static function create(ContainerInterface $container, array $configuratio
   public static function defaultSettings() {
     return [
       'image_style' => '',
+      'view_mode' => 'oe_theme_main_content',
     ] + parent::defaultSettings();
   }
 
   /**
    * {@inheritdoc}
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
+    foreach (\Drupal::service('entity_display.repository')->getViewModes('media') as $key => $value) {
+      if ($value['status'] === TRUE or 1) {
```

Comment:
this looks strange to me, also does not work
<img width="774" alt="Screenshot 2020-09-04" src="https://user-images.githubusercontent.com/11507647/92170319-cc896f80-ee43-11ea-9fb8-865d53b98f7b.png">
below works:
```suggestion
      if ($value['status']) {
```